pegasus/sites.v3/code.org/public/about/careers/engineering.haml

```diff
@@ -0,0 +1,82 @@
+---
+title: Join Our Engineering Team!
+theme: responsive_full_width
+---
+
+%link{href: "/css/generated/design-system-pegasus.css", rel: "stylesheet"}
+%link{href: "/css/generated/page/engineering-styles.css", rel: "stylesheet"}
+
+%section.hero-banner-basic.hero-banner-basic--engineering
+    .wrapper
+        .flex-container
+            .text-wrapper.col-60.flex-container.align-items-center
+                .hero-banner-basic__content
+                    %h1{style: "color: white"}
+                        =hoc_s(:engineering_hero_heading)
+                    %p{style: "color: white"}
+                        =hoc_s(:engineering_hero_desc)
+                    %a{href: "https://code.org/about/careers#roles ", class: "link-button", target: "_blank", rel: "noopener noreferrer", aria:{label: hoc_s("accessibility_explore_title")}}
+                        =hoc_s(:call_to_action_open_positions)
+            .col-40.flex-container.justify-end
+                %img.hero-banner-basic__image{src: "/images/about/careers/engineering/hero-image.png", alt: ""}
+        .clear
+
+
+%section.engineering-team#accessibility-team
+    .wrapper
+        %h2=hoc_s(:engineering_team_title)
+        .action-block.action-block--two-col
+            %img{src: "/images/about/careers/engineering/careers_team.jpg", alt: hoc_s(:accessibility_explore_title), class: "col-50 action-block__image"}
+            .text-wrapper.col-50
+                %p.heading-sm=hoc_s(:engineering_team_card_text_1)
+                %p.heading-sm=hoc_s(:engineering_team_card_text_2)
+                %a{href: "https://code.org/about/careers#roles ", class: "link-button", target: "_blank", rel: "noopener noreferrer", aria:{label: hoc_s("accessibility_explore_title")}}
+                    =hoc_s(:call_to_action_open_positions)
+        .clear
+
+= view :section_divider_line
+
+%section.our-team-text
+    .wrapper
+        %h2=hoc_s(:engineering_our_team_title)
+        %p=hoc_s(:engineering_our_team_desc)
+        .our-team-text__divider.our-team-text__divider--small
+        %div
+            %img.our-team-text__image.our-team-text__image--1{src: "/images/about/careers/engineering/our-team--1.jpg", alt: hoc_s(:engineering_our_team_sub_title_1)}
+            %h3.heading-md=hoc_s(:engineering_our_team_sub_title_1)
+            %p=hoc_s(:engineering_our_team_text_1, markdown: :inline, locals: {youtube_url: "https://www.youtube.com/watch?v=YMjRTjonlWg"})
```

Comment:
Can you add `target: "_blank" rel: "noopener noreferrer"` so this link opens in a new tab? We like to have Code.org URLs open in the same tab, but all external URLs open in a new one.